package/libs/libjson-c/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=json-c
 PKG_VERSION:=0.13.1
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```

Comment:
Remove the package version bump, its not needed.

Reply:
done